rest_framework-stubs/serializers.pyi

```diff
@@ -144,19 +141,16 @@ def as_serializer_error(exc: Exception) -> Dict[str, List[ErrorDetail]]: ...
 class Serializer(
     BaseSerializer[
         Union[_MT, Mapping[str, Any]],
```

Comment:
```suggestion
        Union[_IN, Mapping[str, Any]],
```
_MT is bound to Model so it should be used only with ModelSerializer

Reply:
No need to make this generic this specific, let's use just _IN, someone can create subclass that will work with arbitrary objects not models or dict

Reply:
I disagree.

1. Model serializer is a specialist class for models.
2. We do not need to account for what users might do, but for the drf
3. We need proper type checking